source/Calamari.Tests/KubernetesFixtures/Commands/Executors/GatherAndApplyRawYamlExecutorTests.cs

```diff
@@ -119,6 +119,145 @@ public async Task AppliesKubernetesManifestsAndAddsResourceIdentifiers()
             log.ServiceMessages[1].Properties.Should().Contain(new KeyValuePair<string, string>("name", "CustomResources(basic-service)"));
         }
 
+        [Test]
+        public async Task AppliesKubernetesManifestsWithServerSideApply()
+        {
+            // Arrange
+            AddTestFiles();
+            SetupCommandLineRunnerMocks();
+            var variables = new CalamariVariables
+            {
+                [KnownVariables.EnabledFeatureToggles] = FeatureToggle.GlobPathsGroupSupportFeatureToggle.ToString(),
+                [KnownVariables.OriginalPackageDirectoryPath] = StagingDirectory,
+                [SpecialVariables.CustomResourceYamlFileName] = "dirA/*\ndirB/*",
+                [SpecialVariables.ServerSideApplyEnabled] = "true"
+            };
+            var runningDeployment = new RunningDeployment(variables);
+            var executor = CreateExecutor(variables, fileSystem);
+            var expectedYamlGrouping = $"{Path.Combine(StagingDirectory, "grouped", "1")};{Path.Combine(StagingDirectory, "grouped", "2")}";
+
+            // Act
+            var result = await executor.Execute(runningDeployment, RecordingCallback);
+
+            // Assert
+            result.Should().BeTrue();
+            variables.Get(SpecialVariables.GroupedYamlDirectories).Should().Be(expectedYamlGrouping);
+
+            commandLineRunner.ReceivedCalls().Count().Should().Be(4);
+            var commandLineArgs = commandLineRunner.ReceivedCalls().SelectMany(call => call.GetArguments().Select(arg => arg.ToString())).ToArray();
+            commandLineArgs[0]
+                .Should()
+                .Contain("apply -f")
+                .And.Contain("--recursive")
+                .And.Contain("-o json")
+                .And.Contain($"{Path.Combine("grouped", "1")}")
+                .And.Contain("--server-side")
+                .And.Contain("--field-manager octopus");
```

Comment:
It's probably a good idea to check that it also does not contain the `--force-conflicts` flag since this is the only difference between this test and the next one